src/applications/mhv-secure-messaging/tests/e2e/utils/constants.js

```diff
@@ -223,6 +222,10 @@ export const Locators = {
   HEADERS: {
     DRAFTS_HEADER: '#draft-reply-header',
   },
+  DROPDOWN: {
+    RECIPIENTS: `va-additional-info[trigger^="If you"]`,
```

Comment:
I would recommend to have this be more specific than just `If you`

Reply:
there are two drop downs with same tags and attributes so the text is unique identifier.